lib/ex_doc/formatter/html/search_items.ex

```diff
@@ -27,14 +27,15 @@ defmodule ExDoc.Formatter.HTML.SearchItems do
     encode(
       "#{id}.html##{HTML.text_to_id(header)}",
       "#{title} - #{header}",
+      "#{title} - #{header}",
```

Comment:
The generated search_items.js is already really large, so I don't think we should add a new field. We already have a title field and we should re-use it. Therefore, can you please revert the changes to this file and re-use the existing file on the `.js` side of things? Thank you!